panel/src/App.vue

```diff
@@ -122,10 +122,14 @@ export default {
     fatal(html) {
       if (html !== null) {
         this.$nextTick(() => {
-          let doc = this.$refs.fatal.contentWindow.document;
-          doc.open();
-          doc.write(html);
-          doc.close();
+          try {
+            let doc = this.$refs.fatal.contentWindow.document;
+            doc.open();
+            doc.write(html);
+            doc.close();
+          } catch (e) {
+            console.error(e);
```

Comment:
Shouldn't the original error also be printed to the console in this case so that the issue can be debugged even if the iframe doesn't work? Or would that be too spammy?

Reply:
I think it's too verbose when you send the entire HTML to the console. In a worst case scenario it's a full HTML document.

Reply:
👍